Logger/FileLogger.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+
+namespace Logger;
+
+    public class FileLogger : BaseLogger
+    {
+
+        public string PathName {  get; set; } 
+
+        public FileLogger(string pathName)
+        {
+            PathName = pathName;
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+
+            using StreamWriter streamWrite = File.AppendText(PathName);
+
+            string log = DateTime.Now.ToString(CultureInfo.InvariantCulture) + " " + ClassName + " "
+                + logLevel.ToString() + ": " + message;
+
+            streamWrite.WriteLine(log);
+
```

Comment:
If I remember correctly, we have to close the StreamWriter every time.
 ```suggestion
streamWrite.Close();
```